### DISCUSSION TITLE
Idea - AI voice cloning for new narration and voice overs audio

### DISCUSSION THREAD

**Discussion by ageekhere** [CorsixTH/CorsixTH#2422] [Research Room]
**The idea** 
To use AI voice cloning software to create new narration and voice overs audio.

We can use all of the Announcer's Voices as learning data
https://www.youtube.com/watch?v=tn-U_Yo1Ans

Then find a good AI voice cloning software and train it using this voice data set.

Once trained new narration voice overs could be added, like the mission briefing at the start of the game.

Post your ideas and thoughts.

**Comment by ravenPenfold** [comment]
In my honest opinion, not probably beneficial.

While yes, screen readers are important, and while AI is a good tool (when used in moderation), I don't think using AI to replicate someone's voice (without thier permission) can be justified for 3 reasons:
1. It doesn't make sense in the game lore - in that the receptionist tends to be that who makes the announcements
2. If we did that, then it would be just as bad as replacing Voice Actors and other creators in media with Artificial Intelligence
3. Unless we created said AI for the game (effectively making it Text to Speech) ourselves, and then taught it, we would have to rely on external libraries OR not have narration for custom levels, which would effectively defeat the purpose imo.

**Reply by ageekhere** [reply]
I think the hardest part would be the quality of the voice clone, it would be very hard for it to sound good.

**Comment by MandrakeHorse** [comment]
It is possible that in the distant future, there may be an effort to recreate all the graphics and sounds from scratch (similar to OpenTTD). In the case of the receptionist's announcements, it would be better and simpler to recruit a voice actress for the role.

**Comment by Alberth289346** [comment]
I am not a lawyer, but original sounds are licensed by the original game. Taking them and then making some "new" variations of them is probably insufficient in court to consider the variations free of the previous license.

A much more safe way is indeed to make really new ones instead.
